test/conversations/Conversations.test.ts

```diff
@@ -288,13 +266,11 @@ describe('conversations', () => {
       expect(aliceConvo instanceof ConversationV1).toBeTruthy()
       await bob.publishUserContact(false)
       alice.forgetContact(bob.address)
-      await sleep(100)
```

Comment:
I removed a bunch of these sleeps, which haven't been necessary ever since we made publishing synchronous on the nodes.